core-bundle/contao/forms/FormText.php

```diff
@@ -120,6 +120,7 @@ public function __set($strKey, $varValue)
 				break;
 
 			case 'placeholder':
+			case 'autocomplete':
```

Comment:
Pretty sure this one isn't needed since the magic method would handle this, actually the same for `placeholder`

Reply:
It is needed, otherwise it would be put into `$this->arrConfiguration` instead.

Reply:
When testing this PR, I noticed that `autocomplete` is put into `$this->arrAttributes` even without this change. This is due to the case in:
https://github.com/contao/contao/blob/6f8832ffea3c8454c90d0b09429c5654232afe39/core-bundle/contao/library/Contao/Widget.php#L263
Setting `$widget->autocomplete = false` in `loadFormField` to get `autocomplete="off"` is now no longer supported. I don't know if someone uses this, but just wanted to mention it. You can now remove the attribute by setting it to `false`, which is probably better.

Reply:
`autocomplete="off"` is not the same as as not having an `autocomplete` attribute at all so we should not change that behavior I think. We should keep the mapping for `false` to `"off"` and `true` to `"on"`.

Reply:
Using
```php
case 'autocomplete':
    if (\is_bool($varValue))
    {
	    $varValue = $varValue ? 'on' : 'off';
    }

    $this->arrAttributes[$strKey] = $varValue;
    break;
```
now

@ausi hinted me at the spellchecker within Widget considering the boolean

Reply:
Can’t this change here be removed completely? As the code in the parent class already does exactly the same?

Reply:
🤨 'on' & 'off' is pretty generic isn't it?
There are tons of legitimate values: https://www.w3schools.com/tags/att_input_autocomplete.asp

Reply:
@asaage you are misunderstanding the issue. This PR implements the ability to set these values.